app/controllers/following_controller.rb

```diff
@@ -0,0 +1,45 @@
+class FollowingController < ApplicationController
+  before_action :require_login, :redirect_if_current_user_equals_params
+
+  def follow
+    followee = User.find_by!(login: params[:login])
+
+    if already_following?(followee.id)
```

Comment:
With `@user` set, then you could drop the `find_by! and even the `already_following` method and go with `if current_user.is_following? @user`